src/modules/common/components/__tests__/Failure.spec.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import { Failure } from '../Failure'
+
+describe('Failure', () => {
+  const mockTranslate = jest.fn((msg) => msg)
+
+  test('should match snapshot', () => {
+    const wrapper = shallow(
+      <Failure error='Error Message' t={mockTranslate} />
+    )
+
+    expect(wrapper).toMatchSnapshot()
+  })
+})
```

Comment:
I think, we miss the test, that if we `import Failure from '../Failure'` that the translate(...) has been wrapped around it, right?

Reply:
Our translate mock and the snapshot testing should catch these mistakes

Reply:
But there's no snapshot for that, is there?

Reply:
It creates a snapshot which has the translated key in it. Uhmmm....
To what does our mock translate keys?

So yeah it probably does not test for the translate HOC atm. But do we really need it? I mean we still don't know if the translation exists etc.

Reply:
No, but we know that this.props.t doesnt raise an exception because it isnt there
And There are translated keys, because steff mocked t himself in Failure.spec.js

Reply:
So yeah the test for ConnectedFailure is missing 🔥

Reply:
will be added

Reply:
do you think a snapshot is enough?

Reply:
Or lets call it TranslatedFailure. So you only need to render the translated component.